lib/commands/install.js

```diff
@@ -131,7 +131,7 @@ class Install extends ArboristWorkspaceCmd {
     args = args.filter(a => resolve(a) !== this.npm.prefix)
 
     // `npm i -g` => "install this package globally"
-    if (where === globalTop && !args.length) {
+    if (isGlobalInstall && where === globalTop && !args.length) {
```

Comment:
`where` is already coupled to ` isGlobalInstall`:

```
    const isGlobalInstall = this.npm.global
    const where = isGlobalInstall ? globalTop : this.npm.prefix
```

I suspect we don't need both tested here.